sys-cluster/lmod/lmod-8.4.20.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..3} )
+inherit autotools lua-single
+
+DESCRIPTION="Environment Module System based on Lua"
+HOMEPAGE="https://lmod.readthedocs.io/en/latest"
```

Comment:
Please also add the Github page as 2ndary HOMEPAGE here.

Reply:
I've added it to the `<upstream>` tag in metadata.xml instead. Should be fine too?

Reply:
It doesn't provide a clickable link with tools like `equery` or `pgo` ;)